### PR TITLE
Add pm2 custom metrics

### PR DESCRIPTION
**Pull request by TheDevMinerTV**
I've added custom metrics for pm2 like status, progress, blockchain height, difficulty and net hashrate.

### CODE REVIEW

**Comment by TheDevMinerTV** [comment]
Okay

**Comment by brandonlehmann** [comment]
To be more clear, I mean a peer dependency, not optional in the normal sense of the word.

**Comment by TheDevMinerTV** [comment]
Oh, okay. That was what I was thinking about...
My brain almost gave up :joy:

File: package.json [hunk 1]
```diff
@@ -30,5 +30,6 @@
   },
   "devDependencies": {
     "standard": "^11.0.1"
-  }
+  },
+  "optionalDependencies": {}
```
Comment (brandonlehmann) [inline package.json]: You actually need to specify the package here -- but see:

https://docs.npmjs.com/files/package.json#peerdependencies

to define a peer dependency

**Comment by TheDevMinerTV** [comment]
Like this?

**Comment by brandonlehmann** [comment]
> Like this?

Yup.

I'll pull it down and give it a test later tonight.

**Comment by TheDevMinerTV** [comment]
Okay

Brandon Lehmann <notifications@github.com> schrieb am Do., 23. Mai 2019,
00:18:

**Pull request merged**